controller/handler/compliance-count.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * @module controller/handler/trial
+ */
+
+const database = require('../../model');
+const processPatient = require('../helper/process-patient');
+const processTrial = require('../helper/process-trial');
+const moment = require('moment');
+const getCount = require('../helper/process-compliance-count');
+
+/**
+ * A dashboard with an overview of a specific trial.
+ * @function getComplianceCount
+ * @param {row} currentTrial - Row
+ * @param {row} stages - Row
+ * @param {row} patients - Row
+ * @param {View} reply - Row
+ * @returns {View} Rendered page
+ */
+function getComplianceCount (currentTrial, stages, patients, reply) {
+    const startDate = moment().startOf('Week');
+
+    Promise
+        .all([
+            database.sequelize.query(
+                `
+        select patientId,
+        sum(case when state = 'expired' then 1 else 0 end) as expiredCount
```

Comment:
right, but so does the existing query...
so what you want then is:

``` SQL
SELECT SUM(si.state = 'expired') AS expired, SUM(si.state = 'completed') AS completed
FROM survey_instance AS si
JOIN patient AS pa
ON si.patientId = pa.id
JOIN stage AS st
ON pa.stageId = st.id
WHERE st.trialId = ?
GROUP BY patientId
```

which will count both expired and completed
and will limit the query to just a selected trial